### PR TITLE
Update tedge plugin specs after c8y-mapper change

### PR DESCRIPTION
**Pull request by rina23q**
## Proposed changes
See the comment here. https://github.com/thin-edge/thin-edge.io/pull/2108#discussion_r1310712532

During the work of c8y-mapper, we decided to use the device name inside `tedgeUrl` instead of using "main". So, this PR updates the specs to align with the implementation.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
#2048 

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/src/references/tedge-configuration-management.md [hunk 1]
```diff
@@ -140,12 +140,12 @@ For example, it subscribes to the following topic for the main device.
 tedge mqtt sub 'te/device/main///cmd/config_snapshot/+'
 ```
 
-When a new config snapshot command with the ID "1234" is published by another component, it appears as follows.
+When a new config snapshot command with the ID "1234" for the device name "example" is published by another component, it appears as follows.
```
Comment (didier-wenzek) [inline docs/src/references/tedge-configuration-management.md:143]:
```suggestion
When a new config snapshot command with the ID "1234" is published for the device named "example" by another component, it appears as follows.
```

I would even consider to fully rewrite the sentence.

To start a new config snapshot with the ID "1234" on the device named "example", a component has to publish the following message over MQTT:

File: docs/src/references/tedge-log-management.md [hunk 2]
```diff
@@ -76,12 +76,12 @@ For example, it subscribes to the following topic for the main device.
 tedge mqtt sub 'te/device/main///cmd/log_upload/+'
 ```
 
-A new log file upload command with the ID "1234" is published by another component as below.
+A new log file upload command with the ID "1234" for the device name "example" is published by another component as below.
```
Comment (didier-wenzek) [inline docs/src/references/tedge-log-management.md:79]:
```suggestion
A new log file upload command with the ID "1234" is published for the device named "example" by another component as below.
```

File: docs/src/references/tedge-configuration-management.md [hunk 3]
```diff
@@ -140,12 +140,12 @@ For example, it subscribes to the following topic for the main device.
 tedge mqtt sub 'te/device/main///cmd/config_snapshot/+'
 ```
 
-When a new config snapshot command with the ID "1234" is published by another component, it appears as follows.
+When a new config snapshot command with the ID "1234" is published for the device named "example" by another component, it appears as follows.
```
Comment (rina23q) [inline docs/src/references/tedge-configuration-management.md:143]:
```suggestion
To start a new config snapshot with the ID "1234" on the device named "example", a component has to publish the following message over MQTT:
```

**Review by rina23q — COMMENTED** [review]
Add suggestions from Didier

**Pull request merged**